apps/minifront/src/state/send.ts

```diff
@@ -152,10 +152,11 @@ export const amountMoreThanBalance = (
    */
   amountInDisplayDenom: string,
 ): boolean => {
-  if (asset.balanceView?.valueView.case !== 'knownAssetId')
-    throw new Error('unknown asset selected');
```

Comment:
We filter out unknown assets for ibc and swaps, but I believe there is no reason to disallow sending unknown assets